### PR TITLE
More Capitalization Changes

### PR DESCRIPTION
**Pull request by PEmu1**

### CODE REVIEW

File: Source/Core/DolphinWX/VideoConfigDiag.cpp [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ void VideoConfigDiag::PopulatePostProcessingShaders()
   if (shaders.empty())
     return;
 
-  choice_ppshader->AppendString(_("(off)"));
+  choice_ppshader->AppendString(_("Off"));
```
Comment (PEmu1) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: I wasn't sure why this option had parentheses around it as that doesn't match the style of the interface elsewhere. If there was a reason for doing it it can be changed back.

File: Source/Core/DolphinWX/Config/GeneralConfigPane.cpp [hunk 2]
```diff
@@ -52,9 +52,9 @@ void GeneralConfigPane::InitializeGUI()
   for (const CPUCore& cpu_core : m_cpu_cores)
     m_cpu_engine_array_string.Add(cpu_core.name);
 
-  m_dual_core_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Dual Core (speedup)"));
+  m_dual_core_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Dual Core (Speedup)"));
   m_cheats_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Cheats"));
-  m_force_ntscj_checkbox = new wxCheckBox(this, wxID_ANY, _("Force Console as NTSC-J"));
+  m_force_ntscj_checkbox = new wxCheckBox(this, wxID_ANY, _("Force Console As NTSC-J"));
```
Comment (Parlane) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: Should as be capitalised? You didnt do it for "by" in "Toggle 3D Side-by-Side"

File: Source/Core/Core/HotkeyManager.cpp [hunk 3]
```diff
@@ -12,7 +12,7 @@
 const std::string hotkey_labels[] = {
     _trans("Open"),
     _trans("Change Disc"),
-    _trans("Refresh List"),
+    _trans("Refresh Game List"),
```
Comment (PEmu1) [inline Source/Core/Core/HotkeyManager.cpp]: To match #4630.

File: Source/Core/DolphinWX/Config/GeneralConfigPane.cpp (hunk 2 above)
Comment (PEmu1) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: It's debatable but I believe as usually gets capitalized in a title. See "Subordinating conjunctions": http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html for example

Comment (Parlane) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: That's fair, thanks I wasn't 100% aware of title rules.

File: Source/Core/DolphinWX/Config/GeneralConfigPane.cpp [hunk 4]
```diff
@@ -23,13 +23,13 @@
...
+      {PowerPC::CORE_INTERPRETER, _("Interpreter (Slowest)")},
+      {PowerPC::CORE_CACHEDINTERPRETER, _("Cached Interpreter (Slower)")},
 #ifdef _M_X86_64
-      {PowerPC::CORE_JIT64, _("JIT Recompiler (recommended)")},
-      {PowerPC::CORE_JITIL64, _("JITIL Recompiler (slow, experimental)")},
+      {PowerPC::CORE_JIT64, _("JIT Recompiler (Recommended)")},
+      {PowerPC::CORE_JITIL64, _("JITIL Recompiler (Slow, Experimental)")},
 #elif defined(_M_ARM_64)
-      {PowerPC::CORE_JITARM64, _("JIT Arm64 (experimental)")},
+      {PowerPC::CORE_JITARM64, _("JIT Arm64 (Experimental)")},
```
Comment (JosJuice) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: I'm not sure if these are a good idea (and also the "fast" and "slow" in AudioConfigPane.cpp). They're not part of the names, they're just descriptions.

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp [hunk 5]
```diff
@@ -257,13 +257,13 @@ wxSizer* ControllerConfigDiag::CreateWiimoteConfigSizer()
...
   m_passthrough_sync_text =
-      new wxStaticText(this, wxID_ANY, _("Sync real Wii Remotes and pair them"));
+      new wxStaticText(this, wxID_ANY, _("Sync Real Wii Remotes and Pair Them"));
   m_passthrough_sync_btn =
       new wxButton(this, wxID_ANY, _("Sync"), wxDefaultPosition, wxDLG_UNIT(this, wxSize(60, -1)));
   m_passthrough_sync_btn->Bind(wxEVT_BUTTON, &ControllerConfigDiag::OnPassthroughScanButton, this);
 
   m_passthrough_reset_text =
-      new wxStaticText(this, wxID_ANY, _("Reset all saved Wii Remote pairings"));
+      new wxStaticText(this, wxID_ANY, _("Reset All Saved Wii Remote Pairings"));
```
Comment (JosJuice) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: This one and the one above aren't titles of buttons, they're descriptions that are next to buttons. They should likely be in sentence case.

File: Source/Core/DolphinWX/VideoConfigDiag.cpp (hunk 1 above)
Comment (JosJuice) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: It's probably for making it stand out more from the other options (and there are a lot of options). We do similar things elsewhere, e.g. <Nothing> in the EXI slot config.

File: Source/Core/DolphinWX/Config/GeneralConfigPane.cpp (hunk 2 above)
Comment (Sarkie) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: "Capitalize all words in titles of publications and documents, except a, an, the, at, by, for, in, of, on, to, up, and, as, but, or, and nor."
 from that article @PEmu1

Comment (Parlane) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: "Good question, and one meriting an answer: "As" can be either a subordinating conjunction or a preposition: Capitalize it when it is a subordinating conjunction; do not capitalize it when it is a preposition."

Comment (Parlane) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: So in this case as is infact a preposition and not a subordinating conjunction.

subordinating conjunction example:
As it's raining, I'm staying in.

preposition example:
He works as a waiter.

Comment (Parlane) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: I hate English...

Comment (Sarkie) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: It should be "as" in this case imo and also: http://titlecapitalization.com/ says the same.

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp (hunk 5 above)
Comment (leoetlino) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: Yeah, it would look a bit ridiculous if not in normal case.

File: Source/Core/Core/HotkeyManager.cpp [hunk 6]
```diff
@@ -36,7 +36,7 @@ const std::string hotkey_labels[] = {
     _trans("Start Recording"),
     _trans("Play Recording"),
     _trans("Export Recording"),
-    _trans("Read-only mode"),
+    _trans("Read-Only mode"),
```
Comment (leoetlino) [inline Source/Core/Core/HotkeyManager.cpp]: Isn't read-only considered to be a single word?

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp [hunk 7]
```diff
@@ -227,7 +227,7 @@ wxSizer* ControllerConfigDiag::CreateWiimoteConfigSizer()
 
   auto* const box = new wxStaticBoxSizer(wxVERTICAL, this, _("Wii Remotes"));
 
-  m_passthrough_bt_radio = new wxRadioButton(this, wxID_ANY, _("Passthrough a Bluetooth adapter"),
+  m_passthrough_bt_radio = new wxRadioButton(this, wxID_ANY, _("Passthrough a Bluetooth Adapter"),
```
Comment (leoetlino) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: This isn't really a title, so I find it a bit weird to capitalise adapter.

File: Source/Core/DolphinWX/VideoConfigDiag.cpp (hunk 1 above)
Comment (PEmu1) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: So do you think the change if fine or it should be changed back to having them?

File: Source/Core/DolphinWX/Config/GeneralConfigPane.cpp (hunk 4 above)
Comment (PEmu1) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: Yes I was thinking about that myself but we have made other descriptions capitalized as well such as in speed limit and I wanted consistency. To be honest in this case I would like to delete these altogether and put them in a tooltip instead but I am not sure if others would agree.

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp (hunk 5 above)
Comment (PEmu1) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: Is there any other part of the interface like this? Perhaps a colon could be added?

File: Source/Core/DolphinWX/Config/GeneralConfigPane.cpp (hunk 4 above)
Comment (JosJuice) [inline Source/Core/DolphinWX/Config/GeneralConfigPane.cpp]: Using a tooltip is a bad idea, because it's very hard to discover. The text is short enough that putting it right next to the title like we do right now doesn't lead to any practical problems.

File: Source/Core/DolphinWX/VideoConfigDiag.cpp (hunk 1 above)
Comment (JosJuice) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: I think it's a bit better if we change it back, but I'm fine with having it the new way if someone else thinks that's ideal.

File: Source/Core/Core/HotkeyManager.cpp (hunk 6 above)
Comment (PEmu1) [inline Source/Core/Core/HotkeyManager.cpp]: I think it is fine to capitalize like that: http://www.accu-assist.com/grammar-tips-archive/GrammarTip_capitalization-titles-headings-hyphenated-words.htm but it can probably be debated.

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp (hunk 7 above)
Comment (PEmu1) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: We use that style for other radio button labels in the interface. I don't see why this one would be different.

File: Source/Core/DolphinWX/VideoConfigDiag.cpp (hunk 1 above)
Comment (PEmu1) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: Okay. I am going to leave this for now to see if anyone else has an opinion on it. Personally I do like this change as it looks neater especially when comparing to the stereoscopic 3D dropdown which is in the same tab.

Also if it does get reverted are you okay keeping it capitalized?

Comment (JosJuice) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: I'm not sure. The shaders we ship with use lowercase names, so writing this in uppercase makes it look a bit out of place. But on the other hand, maybe it actually is good that it looks out of place so it's easier to notice that it's a special option.

You have a point about Stereoscopic 3D Mode using Off. I was only thinking of how things looked within one dropdown, not how the whole tab looks when you're not inside a dropdown.

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp (hunk 5 above)
Comment (leoetlino) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: I don't think there should be a colon, since it's very obviously a description for the button next to it, and the other labels in the Controller Configuration dialog don't have a colon either.

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp (hunk 7 above)
Comment (leoetlino) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: I don't know, it just looks odd to me. Feel free to ignore this though, it's just my opinion.

**Comment by lioncash** [comment]
Is there anything left to be addressed on this or whatnot?

**Comment by JosJuice** [comment]
The changes from sentence case to title case in the controllers window should be reverted IMO. I'm less sure about the `(off)`/`Off` thing.

**Comment by PEmu1** [comment]
@lioncash @JosJuice I'm still not 100% sure about using sentence case in the controller screen but I've changed it back as there doesn't seem to be agreement there. I have left Off as it is and would prefer to keep it that way unless someone has strong feelings about it being written differently.

Otherwise I think this should be ready to merge.

File: Source/Core/Core/HotkeyManager.cpp (hunk 6 above)
Comment (ligfx) [inline Source/Core/Core/HotkeyManager.cpp]: This seems wrong to me, since "mode" is still lowercase. The hyphenated parts should only be capitalized if it's in title case.

Comment (PEmu1) [inline Source/Core/Core/HotkeyManager.cpp]: Mode should be capitalized, I think I accidentally missed it the first time around. I will fix that.

**Comment by JosJuice** [comment]
@MayImilae Does this look good to you?

**Comment by MayImilae** [comment]
Wow the OCD in this one. Ok, going through them...

I don't agree with "24-Bit". Lowercase is normal for bit, and uppercase is usually used for byte. 1mb versus 1mB, for example. It should stay lowercase to help reflect that, imo.

As for the rest... meh. It's all just subjective OCD nitpicking. It doesn't really matter.

**Comment by ligfx** [comment]
@Maylmilae if the convention here is to put everything in title case, then capitalizing "24-Bit" is correct (since it's a non-trivial standalone word within a hyphenated word).

**Comment by MayImilae** [comment]
@ligfx It doesn't really matter either way, though. The closest grammar rule would be for lists, and it has no capitalization requirements, so grammar rules don't apply. And the convention is from trying to not conflict with operating system (for right click menus and mac menu bar dropdowns and the like), which doesn't apply here since all of things this touches are entirely contained without Dolphin windows. So there no such thing as "correct" here. 

Basically, there's nothing binding. It's all subjective; these can be anything we want them to be!

Anyway, in my opinion, these changes are a little silly, but fine.

Except for bit, I think that should stay lowercase, to stay true to programming conventions.

**Comment by ligfx** [comment]
Well, sure, all language is subjective, but the thrust of this PR seems to be "make all labels consistently title-case"—as I understand from the various comments referencing title-case rules, if I'm wrong someone please correct me. That's a silly argument that doesn't have a place in engineering, IMO.

Let's step back a bit to why these changes are desired in the first place. Consistency of writing/UX style is good, since it makes things smoother for end-users. The label "Force 24-Bit Color" will look more correct to the majority of English-speaking users than "Force 24-bit Color" will, regardless of how related technical abbreviations may or may not be capitalized.

**Comment by JosJuice** [comment]
> I don't agree with "24-Bit". Lowercase is normal for bit, and uppercase is usually used for byte. 1mb versus 1mB, for example. It should stay lowercase to help reflect that, imo.

That rule is only for acronyms. I wouldn't write byte as Byte in the middle of a normal sentence, for instance.

Writing it as "24-Bit" in the setting name is the way to go in my opinion.

**Comment by PEmu1** [comment]
Yes, b vs. B is for differentiating their symbols. I believe it is just fine to capitalize the word bit when it is written out and I would rather not change that.

**Comment by ligfx** [comment]
@dolphin-emu-bot rebuild

**Comment by PEmu1** [comment]
@JosJuice Are you fine with merging this?

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp (hunk 7 above)
Comment (JosJuice) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: If you change this, you need to change "Emulate the Wii's Bluetooth adapter" below too.

Comment (PEmu1) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: Done.

File: Source/Core/Core/ConfigManager.h [hunk 8]
```diff
@@ -20,7 +20,7 @@ enum class Region;
 }
 
 // DSP Backend Types
-#define BACKEND_NULLSOUND _trans("No audio output")
+#define BACKEND_NULLSOUND _trans("No Audio Output")
```
Comment (JosJuice) [inline Source/Core/Core/ConfigManager.h]: @degasus, is this going to cause any problems? Breaking INI compatibility? (Or do we not care about INI compatibility for this choice?)

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Comment by JosJuice** [comment]
You probably need to rebase on the latest version of master.

**Comment by shuffle2** [comment]
please rebase instead of merging master into the branch
